portindex2json/portindex2json.tcl

```diff
@@ -12,6 +12,43 @@
 
 package require json::write
 
+proc format_string {string} {
+	if {[llength $string] > 1} {
+		foreach element $string {
+			lappend new_string "\"$element\""
```

Comment:
Why not use `::json::write::string`?

Reply:
`::json::write string` is being already used, it would give a single string as output. But we need a list of strings which would be possible only with `::json::write array` and it takes a list as an argument.